app/views/posts/show.html.erb

```diff
@@ -1 +1,14 @@
-<h1>Details of a selected post<h1>
\ No newline at end of file
+<h1>Blog App</h1>
+<article class='no-posts'>
+  <div class='post-title'>
+    <h1><%=@post.title%> by <%=@post.author.name%></h1>
+    <div class='post-count-best'>Comments: <%=@post.comment_counter%>, Likes: <%=@post.likes_counter%></div>
+  </div>
+  <div><%=@post.text%></div>
+  <ul class='comment-content'>
+    <% @post.comments.each do |comment| %>
+      <li><%= comment.author.name %>: <%= comment.text %></li>
+    <% end %>
+    <%= 'No comments' if @post.comments.empty? %>
+  </ul>
+</article>
```

Comment:
- [ ] Kindly make sure to follow the given wireframe for styling the posts's show page. It's nice to have styling, however, please make sure that general layout remains the same. Kindly have a look at the following comparison for better clarity 💡👇


**_Current implementation_**
![localhost_3000_users_1_posts_2](https://github.com/emmanuelsteven/MyBlogApp/assets/106119105/cf603f07-1a41-4751-a03a-e1ffc90e4713) 

**_Given wireframe_**
![image](https://github.com/emmanuelsteven/MyBlogApp/assets/106119105/2959fe8a-e833-4f72-8ac6-aaececc7d4e5)